### ISSUE TITLE
Show last commit day on badges

### ISSUE THREAD

**Issue by chinesedfan** [Bogdanp/awesome-advent-of-code#152]
Because the default last-commit badge will format the date as `December` with the time changes, it is hard to distinguish between repositories now.

I find http://shields.io supports dynamic badges, which means we can design own date format. For example, `YYYY/MM/DD` should be better.

[![Last commit status](https://img.shields.io/badge/dynamic/json.svg?label=last%20commit&url=https%3A%2F%2Frunkit.io%2Fchinesedfan%2Flast-commit-badge%2F1.0.0%3Fuser%3Dchinesedfan%26repo%3Dadventofcode&query=%24.message&colorB=green)](https://github.com/chinesedfan/adventofcode)

What's more, after https://github.com/badges/shields/pull/2473 is merged, we can customize everything besides badge text.

**Comment by Bogdanp** [comment]
I like the `YYYY/MM/DD` format. Feel free to update the python script and submit a PR w/ that change.

**Comment by lorenzleutgeb** [comment]
No, please don't do `YYYY/MM/DD`. [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) is the way to go. Do `YYYY-MM-DD`!

Here, have [this xkcd](https://xkcd.com/1179/):
![xkcd: ISO 8601](https://user-images.githubusercontent.com/542154/70163958-9f473f80-16c0-11ea-80c5-d9495c546bf6.png)

**Comment by chinesedfan** [comment]
> I am afraid you need to deploy the same endpoint to a more reliable server.

@Bogdanp Now an alternative solution is Github actions. I have created a [branch](https://github.com/chinesedfan/awesome-advent-of-code/tree/badge-actions) in my fork to generate badges automatically. The main process is,

- listen to push events (can also be scheduled by crontab, i.e. everyday in December and weekly in other months)
- parse README.md and collect repositories (I focused on Javascript repos now)
- fetch last commit time and generate badges with YYYY-MM-DD format and different colors

Here is a [sample result](https://github.com/chinesedfan/awesome-advent-of-code/blob/js-badge/README-badge.md). If you don't oppose scripts in Javascript, I can send a PR.
